standard/Standard.cmake

```diff
@@ -2,12 +2,11 @@
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
 
 # Need these for the rest of these scripts to work properly
-include(CTest)
```

Comment:
Standard must be included before project() so we can use standard_project instead, and CTest must be included after the first call to project(). I moved this include line to happen inside of standard_project.